vcd/resource_vcd_snat.go

```diff
@@ -40,7 +40,7 @@ func resourceVcdSNAT() *schema.Resource {
 				Optional:     true,
 				Default:      "org",
 				ForceNew:     true,
-				ValidateFunc: validation.StringInSlice([]string{"org"}, false),
+				ValidateFunc: validation.StringInSlice([]string{"ext"}, false),
```

Comment:
Let's allow both "org" and "ext". We're leaving "org" undocumented for now, but we can do some useful tests even with existing code.

Reply:
And what about DNAT resource?